### PR TITLE
add entry for source center engine

### PR DESCRIPTION
**Pull request by Yana-Guseva**

### CODE REVIEW

File: sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    @Column(name = "SCE_ID")
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    @ManyToOne(fetch = FetchType.LAZY)
```
Comment (olegshulyakov) [inline sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java]: Nice that I have not forget about it

File: sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    public Date getLastUpdateTime() {
+        return lastUpdateTime;
+    }
+
+    public void setLastUpdateTime(Date lastUpdateTime) {
+        this.lastUpdateTime = lastUpdateTime;
+    }
+
+    @Column(name = "ACTIVE_FLAG")
+    public Boolean getActiveFlag() {
```
Comment (olegshulyakov) [inline sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java]: Forgot to use `@Type(type = "yes_no")` here. Active flag is stored as CHAR(1)

**Comment by olegshulyakov** [comment]
Please do not miss [automatic review](https://www.codacy.com/app/sedc/stock-exchange-data-collector/pullRequest?prid=561499)

File: sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        SourceCenterEngine that = (SourceCenterEngine) o;
+        return Objects.equals(id, that.id);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id);
```
Comment (Yana-Guseva) [inline sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java]: Can I use name and lastUpdateTime for this prupose?

File: sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Cacheable
+@Table(name = "SOURCE_CENTER_ENGINE")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+public class SourceCenterEngine {
+    private Long id;
```
Comment (olegshulyakov) [inline sedc-core/src/main/java/com/sedc/core/model/SourceCenterEngine.java]: add new line between `public class` and field

**Pull request merged**